src/socket.rs

```diff
@@ -605,11 +621,18 @@ impl UtpSocket {
                     if !use_user_timeout ||
                        ((now - self.last_congestion_update) >= congestion_timeout) {
                         self.last_congestion_update = now;
-                        try!(self.handle_receive_timeout());
+                        if let Err(e) = self.handle_receive_timeout() {
+                            println!("Could m-maybe be `{:?}`", e);
```

Comment:
Oh, I forgot to remove the `println!`. Gonna leave it for now, but when you're finished reviewing, ping me so I can remove them just before you merge the PR.